Framework/Kernel/inc/MantidKernel/case_insensitive_map.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CASE_INSENSITIVE_MAP_H
```

Comment:
Yes, seems like a silly decision in hindsight. Already made the change, just checking it builds.